Casbin.UnitTests/PersistTests/WatcherTest.cs

```diff
@@ -55,13 +55,13 @@ public class SampleWatcher : IWatcher
 
         public void SetUpdateCallback(Func<Task> callback) => _asyncCallback = callback;
 
-        public void Update()
+        public void Update(IWatcherMessage watcherMessage)
```

Comment:
@sociometry don't invent new API by yourself. Follow the Golang's Redis WatcherEx design: https://github.com/casbin/redis-watcher/commit/9447bc35ede29dd2e62c23de5afb59ff8feea670